### DISCUSSION TITLE
Why does Silk.NET.Assimp exist?

### DISCUSSION THREAD

**Discussion by blair-ahlquist** [dotnet/Silk.NET#1643] [General] [answered]
Assimp already exists in C# outside of Silk.NET, so why is there a version here? I think this should be documented somewhere (easy to find).

**Comment by Perksey** [comment, marked as answer]
Because we wanted it to exist? In addition, Silk.NET has documented its existence as much as it has any of its other bindings

![image](https://github.com/dotnet/Silk.NET/assets/11160611/ec27f1da-fa9b-4259-b452-d88d95ce30e0)

Its existence being documented elsewhere outside of the project is none of my business.

Hope this helps.

**Comment by kimkulling** [comment]
I just can talk for me, but I am really happy that you are using the Asset-Importer-Lib. It helps a lot to hardening the public API.

**Reply by Perksey** [reply]
Thanks! And thank you for all your hard work :)

**Reply by kimkulling** [reply]
You are welcome :-)
